### PR TITLE
Add more community members

### PR DESCRIPTION
**Pull request by mattwelborn**
This PR adds many contributors to the team page. My hope is that it can be the single source of truth for generating our Thank You slide.

# TODO:

- [ ] Find names and affiliations where there are TODOs.
- [ ] Did I leave anyone out?
- [ ] Can we shuffle the names within each section? or alphabetize them?
- [ ] Make github links show up

### CODE REVIEW

File: data/team.yml [hunk 1]
```diff
@@ -77,51 +75,328 @@ designTeam:
...
+      designation : "Code contributor"
+      content : "Molpro and entos harness implementation in QCEngine. Contributions to QCSchema."
+      socialIcon :
+        - icon : "tf-ion-social-github"
+          URL : "https://github.com/sjrl"
+
+    - image : "images/empty_px_placeholder.png"
+      name : "Logan Ward"
+      designation : "Code contributor"
+      content : "MPI implementation in QCFractal and QCEngine. TODO"
```
Comment (dgasmith) [inline data/team.yml]: Remove TODO?

File: data/team.yml [hunk 2]
```diff
@@ -77,51 +75,328 @@ designTeam:
...
-      name: "Sebastian Lee"
-      designation: "QCEngine contributor"
-      content: "Molpro wrapper implementation in QCEngine."
-      socialIcon:
-        - icon: "tf-ion-social-github"
-          URL: "https://github.com/sjrl"
+    - image : "images/empty_px_placeholder.png"
+      name : "Sebastian Lee"
+      designation : "Code contributor"
+      content : "Molpro and entos harness implementation in QCEngine. Contributions to QCSchema."
```
Comment (dgasmith) [inline data/team.yml]: Is it "Entos" or "entos"?

File: data/team.yml [hunk 3]
```diff
@@ -77,51 +75,328 @@ designTeam:
...
+    - image : "images/empty_px_placeholder.png"
+      name : "Logan Ward"
+      designation : "Code contributor"
+      content : "MPI implementation in QCFractal and QCEngine. TODO"
+      socialIcon :
+        - icon : "tf-ion-social-github"
+          URL : "https://github.com/WardLT"
+
+    - image : "images/empty_px_placeholder.png"
+      name : "eljost (TODO)"
```
Comment (dgasmith) [inline data/team.yml]: No idea here either.

File: data/team.yml [hunk 4]
```diff
@@ -77,51 +75,328 @@ designTeam:
...
+    - image : "images/empty_px_placeholder.png"
+      name : "Bradley Dice"
+      designation : "QCEngine contributor"
+      content : "Code cleanup in QCEngine."
+      socialIcon :
+        - icon : "tf-ion-social-github"
+          URL : "https://github.com/bdice"
+
+    - image : "images/empty_px_placeholder.png"
+      name : "ahurta92 (TODO)"
```
Comment (dgasmith) [inline data/team.yml]: Adrian Hurtado

File: data/team.yml (hunk 2 above)
Comment (mattwelborn) [reply data/team.yml]: _entos_

**Pull request closed without merging**